gnovm/tests/files/print1.gno

```diff
@@ -0,0 +1,6 @@
+package main
+
+func main() {
+	var a []string
+	println(a)
+}
```

Comment:
It would be good to include expected output like other file tests do
https://github.com/gnolang/gno/blob/master/gnovm/tests/files/a1.gno#L14-L19

Reply:
updated. Thanks :)